hotsos/core/ycheck/events.py

```diff
@@ -275,13 +329,14 @@ def __init__(self, *args, **kwargs):
         # resetting the registry prior to each run and we will therefore need
         # to load searches each time which is why we do this here. This is
         # therefore not intended to be used outside of a test scenario.
-        if len(self.global_searcher) == 0:
+        label = EventsSearchPreloader.__name__
+        if self.global_searcher.is_loaded(label) == 0:
```

Comment:
Don't seem to need the '== 0' here, is_loaded() should return a bool and other callers don't

Reply:
ugh well spotted again. this is the result of a last minute chance on my part.